### PR TITLE
Final items in Issue #251 that aren't covered in other PRs.

### PR DESCRIPTION
**Pull request by cwant**
Included:

* `derived_from`, which points to [dcterms:source](https://github.com/ualbertalib/metadata/blob/master/data_dictionary/profile_generic.md#dctermssource)
* `is_version_of`, which points to [dcterms:isVersionOf](https://github.com/ualbertalib/metadata/blob/master/data_dictionary/profile_generic.md#dctermsisversionof)
* `related_link`, which points to [dcterms:relation](https://github.com/ualbertalib/metadata/blob/master/data_dictionary/profile_generic.md#dctermsrelation)
* `alternative_title`, which points to [dcterms:alternative](https://github.com/ualbertalib/metadata/blob/master/data_dictionary/profile_generic.md#dctermsalternative)

But wait, also included:
* switch predicate from `dcterms` vocabulary to `dc` for [rights]().
* make [creators](https://github.com/ualbertalib/metadata/blob/master/data_dictionary/profile_generic.md#dccreator) required and update seeds/tests.

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -30,10 +30,14 @@ class Item < JupiterCore::LockedLdpObject
...
   has_attribute :embargo_end_date, ::RDF::Vocab::DC.available, type: :date, solrize_for: [:sort]
   has_attribute :license, ::RDF::Vocab::DC.license, solrize_for: [:search]
-  has_attribute :rights, ::RDF::Vocab::DC.rights, solrize_for: :exact_match
+  has_attribute :rights, ::RDF::Vocab::DC11.rights, solrize_for: :exact_match
   # `type` is an ActiveFedora keyword, so we call it `item_type`
   # Note also the `item_type_with_status` below for searching, faceting and forms
   has_attribute :item_type, ::RDF::Vocab::DC.type, solrize_for: :exact_match
+  has_attribute :source, ::RDF::Vocab::DC.source, solrize_for: :exact_match
+  has_multival_attribute :is_version_of, ::RDF::Vocab::DC.isVersionOf, solrize_for: :exact_match
+  has_attribute :alternative, ::RDF::Vocab::DC.alternative, solrize_for: :search
```
Comment (murny) [inline app/models/item.rb]: Is there freedom here to make some of these attributes have better naming? alternative? alternative to what? this isn't clear.

Can we call this `alternative_title`? Or we just do what were told here?

Comment (cwant) [reply app/models/item.rb]: Good idea

File: app/models/item.rb [hunk 2]
```diff
@@ -30,10 +30,14 @@ class Item < JupiterCore::LockedLdpObject
...
+  has_attribute :relation, ::RDF::Vocab::DC.relation, solrize_for: :exact_match
```
Comment (murny) [inline app/models/item.rb]: Maybe this one too? UI this is `related item` not sure if relation indicates that very well.

Comment (cwant) [reply app/models/item.rb]: Thanks Shane, I've pushed some changes and updated the description of this PR to reflect the changes.

**Pull request merged**